docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -191,19 +191,20 @@ func (w *World) RegisterComponents(components ...AnyComponentType) error
 package main
 
 import (
-    "log"
 
-    "github.com/my-username/my-world-engine-project/component"
+"log"
+"github.com/my-username/my-world-engine-project/component"
+"pkg.world.dev/world-engine/cardinal/ecs"
+)"pkg.world.dev/world-engine/cardinal/ecs"
+
 )
 
```

Comment:
this looks a bit messed up

Reply:
goland folded that so I couldn't see it.